src/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+} from 'react-router-dom';
+import './styles/App.css';
+import Main from './pages/Main';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/225399951-a609ddab-a1af-4919-9e27-e0fd54318e0d.png)